components/generic/CookieDisclaimer.vue

```diff
@@ -118,6 +118,9 @@
       justify-content: center;
       display: flex;
       cursor: pointer;
+      background-color: transparent;
+      color: $white;
+      border: none;
```

Comment:
Maybe it is worth adding a class to the buttons-links.scss for a "btn-transparent" with the transparent background-color and the border set to none. So it can be used elsewhere, for example in the MediaActionBar.

Reply:
Done